kart/gui/diffviewer.py

```diff
@@ -323,7 +323,11 @@ def fillTree(self):
             changes = {feat["id"]: feat for feat in changes}
             usedids = []
             for feat in changes.values():
-                changetype, featid = feat["id"].split("::")
+                try:
```

Comment:
need a comment here explaining what it's for and when we can remove it.

Reply:
Good idea, will do this morning.